docs/known_issues.rst

```diff
@@ -75,6 +75,47 @@ Both will throw an exception if units do not cancel, e.g.::
 
 See: https://github.com/astropy/astropy/issues/7582
 
+Multiplying a `pandas.Series` with an `~astropy.units.Unit` does not produce a |Quantity|
+-----------------------------------------------------------------------------------------
+
+Quantities may work with certain operations on `~pandas.Series` but
+this behaviour is not tested.
+For example, multiplying a `~pandas.Series` instance
+with a unit will *not* return a |Quantity|. It will return a `~pandas.Series`
+object without any unit::
+
+.. doctest-requires:: pandas>=1.5
+
+   >>> import pandas as pd
+   >>> import astropy.units as u
+   >>> a = pd.Series([1., 2., 3.])
+   >>> a * u.m
+   0    1.0
+   1    2.0
+   2    3.0
+   dtype: float64
+
+One way around this is to use the in-place shift or shift operator::
+
+.. doctest-requires:: pandas>=1.5
+
+   >>> a << u.m
+   <Quantity [1., 2., 3.] m>
+   >>> a <<= u.m
+   >>> a
+   <Quantity [1., 2., 3.] m>
+
+But this is fragile as this may stop working in future versions of
+pandas if they decide to override the dunder methods. What should be
+secure, though, is to initialize the |Quantity| directly::
+
+.. doctest-requires:: pandas>=1.5
+
+    >>> u.Quantity(a, u.m)
```

Comment:
The previous example converted `a` to a `Quantity`, so `pandas` is not involved here at all.